Hosted/Client/wwwroot/index.html

```diff
@@ -11,7 +11,7 @@
 </head>
 
 <body>
-    <app>Loading...</app>
+    <div id="app">Loading...</div>
```

Comment:
What is the reason for this?

I have a Blazor sample at https://github.com/grpc/grpc-dotnet/tree/master/examples/Blazor. Should it be updated to match?

Reply:
We updated our templates to no longer use the `app` because it's considered deprecated.